opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/ServletFilterSpanDecorator.java

```diff
@@ -105,11 +105,11 @@ public void onError(HttpServletRequest httpServletRequest, HttpServletResponse h
         public void onTimeout(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse,
                               long timeout, Span span) {
             Tags.ERROR.set(span, Boolean.TRUE);
-
             Map<String, Object> timeoutLogs = new HashMap<>();
             timeoutLogs.put("event", Tags.ERROR.getKey());
```

Comment:
Actually do we want to classify this as an error?

Reply:
Maybe `event: timeout` and timeout value are enough.

Reply:
Sounds like a discussion for the OT community, as it should be consistent across all instrumentations?

Reply:
https://github.com/opentracing/specification/blob/master/semantic_conventions.md#log-fields-table in the event there is "timed out"

Reply:
Please comment on #38